docs/uml/macros.iuml

```diff
@@ -312,7 +309,16 @@ skinparam defaultMonospacedFontName Andale Mono
     !endif
 !endif
 
+!ifdef SHOW_NETWORK_GENEALOGY
```

Comment:
I think this stuff can just be part of `!ifdef SHOW_NETWORK_INTERNAL` instead of creating a new flag